src/api/server.rkt

```diff
@@ -33,20 +34,33 @@
          start-job-server
          write-results-to-disk
          *demo?*
-         *demo-output*)
+         *demo-output*
+         _create-job0
+         translate-job)
 
 (define *demo?* (make-parameter false))
 (define *demo-output* (make-parameter false))
 
 ; verbose logging for debugging
-(define verbose #f) ; Maybe change to log-level and use 'verbose?
+(define verbose #t) ; Maybe change to log-level and use 'verbose?
 (define (log msg . args)
   (when verbose
     (apply eprintf msg args)))
 
 ;; Job object, What herbie excepts as input for a new job.
 (struct herbie-command (command test seed pcontext profile? timeline-disabled?) #:prefab)
 
+; action-type 'herbie-command or 'translation
+(struct server-action (action-type associated-type) #:prefab)
+
+(struct translate-job (fpcore to-language) #:prefab)
+
+(define (_create-job0 server-action-type associated-type)
```

Comment:
Why does this have to have such an ugly name? Why not just `create-job`?

Reply:
Also why does this function even exist? Its whole goal is just creating a `server-action` struct?

Reply:
And also why is the user of `server.rkt` responsible for knowing which actions go to the thread and which ones are done immediately?